Reduce size of nu binary from 22.7 -> 16.6 MiB (on x86_64 with default feature).

diff --git a/Cargo.toml b/Cargo.toml
index 26df23417..41773fc58 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -98,7 +98,8 @@ database = ["nu-command/database"]
 [profile.release]
 opt-level = "s" # Optimize for size
 strip = "debuginfo"
-lto = "thin"
+lto = "fat"
+codegen-units = 1
 
 # build with `cargo build --profile profiling`
 # to analyze performance with tooling like linux perf
